### PR TITLE
Remove unstable references, removed wrapper div and update dependencies

### LINKED ISSUE #130: BlockerFunction has `unstable_` removed  react-router-dom@v6.21.3
Can you please make `react-router-prompt` types compatible with react-router-dom v6.21.3?

new version has `unstable_` removed

### PR DESCRIPTION
**Pull request by erictompkins**
The core reason for the update is to remove the wrapper `div` around the content. We're using a grid layout and using ReactModal to show the content. In this case the content is displayed out of the root directory, but the `div` that you have around the content affects our grid layout.

Change the `div` to a fragment solves that.

I also updated the packages and updated the `unstable_` references to their current objects.

### CODE REVIEW

File: src/hooks/use-prompt.ts [hunk 1]
```diff
@@ -2,8 +2,8 @@ import { useCallback, useEffect } from "react"
 import {
   useBeforeUnload,
   useBlocker,
-  unstable_Blocker as Blocker,
-  unstable_BlockerFunction as BlockerFunction,
+  Blocker,
+  BlockerFunction as BlockerFunction,
```
Comment (sshyam-gupta) [inline src/hooks/use-prompt.ts]:
```suggestion
  BlockerFunction,
```

File: src/index.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React, { useCallback } from "react"
-import { unstable_BlockerFunction as BlockerFunction } from "react-router-dom"
+import { BlockerFunction as BlockerFunction } from "react-router-dom"
```
Comment (sshyam-gupta) [inline src/index.tsx]:
```suggestion
import { BlockerFunction } from "react-router-dom"
```

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-router-prompt",
-  "version": "0.6.1",
+  "version": "0.7.1",
+  "name": "@aptuitiv/react-router-prompt",
```
Comment (sshyam-gupta) [inline package.json]: Please keep the name and URL same in package json

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-router-prompt",
-  "version": "0.6.1",
+  "version": "0.7.1",
```
Comment (sshyam-gupta) [inline package.json]:
```suggestion
  "version": "0.7.0",
```

File: package-lock.json
Comment (sshyam-gupta) [inline package-lock.json:1]: Using pnpm here, package-lock is not required. Can you please use pnpm to install the packages which will update pnpm-lock.yaml instead?
https://pnpm.io/installation

File: package.json (hunk 3 above)
Comment (erictompkins) [reply package.json]: I didn't expect you to reply so quickly so I deployed this under my own package to use in our code. Thank you for reviewing so quickly.

**Comment by erictompkins** [comment]
@sshyam-gupta All of your requested changes have been made. Thank you!

**Review by sshyam-gupta — APPROVED** [review]

**Pull request merged**